src/Application/Components/Shop/Shop.jsx

```diff
@@ -1,20 +1,25 @@
 import React from 'react';
-import KeyBoundShopItem from './KeyBoundShopItem';
 import _ from 'underscore';
 import { assetDefaults } from './../../../util/constants';
 
+import KeyBoundShopItem from './KeyBoundShopItem';
+
 const Shop = React.createClass({
   propTypes: {
+    areStatsVisible: React.PropTypes.bool.isRequired,
     assets: React.PropTypes.array.isRequired,
     onItemClick: React.PropTypes.func.isRequired,
+    onToggle: React.PropTypes.func.isRequired,
   },
 
   render() {
     return (
         <div className="panel panel-primary">
 
-          <div className="panel-heading">
-            <h3 className="panel-title broker-title text-center">Real Estate</h3>
+          <div className="panel-heading text-center">
+            <h3 className="panel-title broker-title">Real Estate</h3>
+            <br />
```

Comment:
i strongly prefer to do line breaks as part of the css but meh i won't die on that hill.  with css / styling a lot of the time it's like "well if it looks right"